DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs

```diff
@@ -395,13 +396,6 @@ internal void Validate(bool isModify = false)
             if (this._embeds.Count > 10)
                 throw new ArgumentException("A message can only have up to 10 embeds.");
 
-            if (isModify)
-            {
-                if (this.ReplyId.HasValue)
-                    throw new ArgumentException("You cannot change the ReplyID when modifying a message");
-            }
-            else
-            {
                 if (this.Files?.Count == 0 && string.IsNullOrEmpty(this.Content) && (!this.Embeds?.Any() ?? true) && this.Sticker is null)
```

Comment:
If the surrounding scope bracket was remvoed, this should be unindented. Am I missing something?